modular_skyrat/master_files/code/modules/client/preferences/_preference.dm

```diff
@@ -95,15 +108,53 @@
 	var/datum/sprite_accessory/default_accessory_type
 	/// Path to the corresponding /datum/preference/toggle to check if part is enabled.
 	var/datum/preference/toggle/type_to_check
+	/// Generates icons from the provided mutant bodypart for use in the selection box.
```

Comment:
Could be useful to mention what selection box you're talking about. Also, if it's an icon used in the prefs menu, just ensure it gets cached properly like all the others do.